package.json

```
@@ -25,7 +25,6 @@
     "storybook-start": "npm run storybook --prefix packages/insomnia-components",
     "storybook-legacy-start": "npm run storybook --prefix packages/insomnia-app",
     "preinstall": "node ./scripts/check-version.js",
-
     "app-build:smoke:core": "cross-env SMOKE_TEST=core npm run app-build",
     "app-build:smoke:designer": "cross-env SMOKE_TEST=designer npm run app-build",
     "test:smoke:core": "npm run test:build:core --prefix packages/insomnia-smoke-test",
```

I think this and the ones below needs to be changed to `test:<app>:build`?

```suggestion
    "test:smoke:core": "npm run test:core:build --prefix packages/insomnia-smoke-test",
```

---

Still working on this naming 😅 thanks